en/index.md

```diff
@@ -322,6 +323,118 @@ permitting users or customers to conduct certain activities.
+
+### **Using UI elements as the English words**
+
+**Don’t use UI elements like they’re English words**
+
+For example:
+
+- **Use**: When you’re finished, click **Save** to save your settings.
+- **Don’t use**: When you’re finished, **Save** your settings.
+
+Using the UI text as English words is less clear and may not transfer well to certain translated versions.
+
+### **Click vs. Tap vs. Select**
+
+**Use “Click” for buttons that you click on (or “Tap” if the product is primarily for mobile devices)**
+
+**Use “Select” for selecting from multiple options (e.g., a dropdown menu, multiple menu items, etc.)**
+
+**Use “Select” when there’s multiple instructions separated by > (e.g., Select Preferences > Languages)**
+
+For example:
+
+- **Use**: Click **Settings** to open your user settings.
+- **Use**: Select the machines you want to register, then click **Save**.
+- **Use**: Tap the application to open it.
+- **Don’t use**: Select **Add bookmark** to save your bookmark.
+
+“Click”/”Tap” is preferred over “Select” for UI elements that are definitive. This is because “Select” has an open-ended connotation, while “Click”/”Tap” is direct and definitive. Although “Click”/”Tap” may technically be wrong in certain situations (e.g., if a user is using the mobile version of a web page on their desktop computer), it’s important to consider accessibility and the primary audience of the documentation over these edge cases. Differentiating when users may have to *select* from options or *select* more than one option can be useful for them, especially when following a longer how-to guide (e.g., Click **X**, Click **Y**, Click **Z**, Select the settings you want to apply, Click **Save**). This extra level of precision can also be helpful for non-native English users and those using translated documentation.
+
+### **Bold vs. Italics**
+
+**Use bold for UI elements the user clicks/selects, and quotes or quotes with italics when drawing attention to a specific word or phrase, or using the name of a word as a word**
+
+For example:
+
+- **Use**: Click **Save**
+- **Use**: In the Computers column, click **Register a new computer**.
+- **Use**: Click the link in the text *“You can register new computers by following these instructions”*.
+- **Use**: Use the word “and” instead of “or”*.*
+- **Use**: Use the word *“and”* instead of *“or”*.
+- **Don’t use**: Once you’ve made your selections, click *Save*.
+
+Bolding UI elements can help make the documentation easier to scan for critical information. This is especially good for users who aren’t reading the documentation for the first time and just want key information without having to sift through extraneous documentation.
+
+### **Angled brackets**
+
+**You can use a right angled bracket > for navigating menu items with multiple steps**
```

Comment:
well, i think you get the idea 
 - word wrap the text (makes it so much easier for future diffs)
 -  Stop using bold in the headings, they are already bold
 -  Stop using bold for entire lines, that isn't the style we have been using.

Reply:
Done

Reply:
:medal_sports: